keep/api/models/db/migrations/versions/2025-03-11-16-54_16309df224d1.py

```diff
@@ -0,0 +1,193 @@
+                WHERE conrelid = 'alertenrichment'::regclass 
+                AND conname = 'alert_fingerprint';
+            """
+        ).fetchone()
+        with op.batch_alter_table("alertenrichment") as batch_op:
+            if constraint_exists:
+                batch_op.drop_constraint("alert_fingerprint", type_="unique")
+
+            batch_op.execute(
+                """
+                    WITH duplicates AS (
+                        SELECT id,
+                            ROW_NUMBER() OVER (
+                                PARTITION BY tenant_id, alert_fingerprint 
+                                ORDER BY timestamp DESC
+                            ) AS rn
+                        FROM alertenrichment
+                    )
+                    DELETE FROM alertenrichment
+                    WHERE id IN (SELECT id FROM duplicates WHERE rn > 1);
+                """
+            )
+            batch_op.create_unique_constraint(
+                "uc_alertenrichment_tenant_fingerprint",
+                ["tenant_id", "alert_fingerprint"],
+            )
+    elif dialect == "sqlite":
+        op.create_table(
+            "alertenrichment_new",
+            sa.Column("enrichments", sa.JSON(), nullable=True),
+            sa.Column("id", sqlmodel.sql.sqltypes.types.Uuid(), nullable=False),
+            sa.Column("tenant_id", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+            sa.Column("timestamp", sa.DateTime(), nullable=False),
+            sa.Column(
+                "alert_fingerprint", sqlmodel.sql.sqltypes.AutoString(), nullable=False
+            ),
+            sa.ForeignKeyConstraint(
+                ["tenant_id"],
+                ["tenant.id"],
+            ),
+            sa.PrimaryKeyConstraint("id"),
+        )
+
+        # Copy existing data
+        op.execute(
+            """
+                INSERT INTO alertenrichment_new (id, tenant_id, alert_fingerprint, timestamp, enrichments)
+                SELECT id, tenant_id, alert_fingerprint, timestamp, enrichments FROM alertenrichment;
+            """
+        )
+        op.execute(
+            """
+                WITH duplicates AS (
+                    SELECT id,
+                        ROW_NUMBER() OVER (
+                            PARTITION BY tenant_id, alert_fingerprint 
+                            ORDER BY timestamp DESC
+                        ) AS rn
+                    FROM alertenrichment_new
+                )
+                DELETE FROM alertenrichment_new
+                WHERE id IN (SELECT id FROM duplicates WHERE rn > 1);
+            """
+        )
+        with op.batch_alter_table("alertenrichment_new") as batch_op:
+            batch_op.create_unique_constraint(
+                "uc_alertenrichment_tenant_fingerprint",
+                ["tenant_id", "alert_fingerprint"],
+            )
+
+        op.execute("DROP TABLE alertenrichment;")
+        op.execute("ALTER TABLE alertenrichment_new RENAME TO alertenrichment;")
```

Comment:
May I ask to leave `alertenrichment_old` permanently in the DB and remove it in 3-4 migrations later once we're sure it works fine?